frontends/mit-open/src/pages/AboutPage/AboutPage.tsx

```diff
@@ -120,126 +111,133 @@ const AboutPage: React.FC = () => {
-                  much of it for free
+                  Learn for free from the same materials used by MIT students on
+                  campus
                 </li>
                 <li>
-                  Build job-relevant skills through application-focused
-                  offerings{" "}
+                  Receive a certificate from MIT or learn for your own enjoyment
                 </li>
                 <li>
-                  Experience peer-to-peer interactions and grow your network
+                  Earn program credentials and apply for an accelerated master's
+                  degree program at MIT
                 </li>
                 <li>
-                  Receive a certificate from MIT, or just learn for your own
-                  enjoyment
+                  Download, share, and modify thousands of learning resources
                 </li>
+                <li>Access content from world-renowned faculty and experts</li>
                 <li>
-                  Learn from the same materials used by MIT students on campus
+                  Build job-relevant skills through application-focused
+                  offerings
                 </li>
                 <li>
-                  Download, share and modify thousands of learning resources
+                  Experience peer-to-peer interactions and grow your network
                 </li>
                 <li>Continue your education at your own pace</li>
-              </ul>
+              </List>
             </SubHeaderTextContainer>
             <SubHeaderImage />
           </SubHeaderContainer>
           <BodySection>
-            <BodyText variant="h4" id={WHAT_IS_MIT_OPEN_FRAGMENT_IDENTIFIER}>
-              What is MIT Open?
-            </BodyText>
-            <BodyText variant="body1">
-              MIT Open is the best place to explore all of MIT's non-degree
-              learning programs. Learners can compare offerings across
-              departments and formats and complete a profile to receive
-              personalized recommendations that match their interests and goals.
-              <br />
-              <br />
-              MIT Open makes it easy to stay up-to-date on the latest non-degree
-              learning resources from MIT. Subscribe to topics, units, and
-              departments of interest to be notified when new courses, programs,
-              and more are added.
-              <br />
-              <br />
-              This portal includes courses and programs from MITx, MIT
-              Bootcamps, MIT OpenCourseWare, MIT Professional Education, MIT
-              Sloan Executive Education, MIT xPRO, and other departments across
-              MIT.
-            </BodyText>
+            <Typography
+              variant="h4"
+              component="h2"
+              id={WHAT_IS_MIT_OPEN_FRAGMENT_IDENTIFIER}
+            >
+              What is {process.env.SITE_NAME}?
```

Comment:
Good call with using `SITE_NAME` here. Can we wrap this section in the white background styles from the design?